src/get-kinde-session.js

```diff
@@ -236,7 +225,21 @@ export const getKindeSession = async (request) => {
     }
   };
 
+  const refreshTokens = async () => {
+    try {
+      await kindeClient.refreshTokens(sessionManager);
+      const headers = generateCookieHeader(request, cookies);
+      return headers;
+    } catch (error) {
+      console.error(error);
+      return null;
+    }
+  };
+
+  const headers = await refreshTokens();
```

Comment:
I’m not sure I understand the purpose of this.

In remix.js, you create a kindeSession in each protected loader or action. I believe refreshTokens should only be called when the access token expires; otherwise, session verification would unnecessarily increase response time.